Dockerfile

```diff
@@ -4,8 +4,7 @@
 ########################################################################################################################
 ########################################################################################################################
 
-# Switched from, node:22-bookworm because node 22.5.0 didn't build
-FROM node:22.4.1 AS updated-local
+FROM node:22-bookworm AS updated-local
```

Comment:
waarom ook een andere versie?

Reply:
Dat had ik in eerste instantie geprobeerd om te checkenn of deze base images het wel toelaat om het certificaat te verifiëren. Dat was niet de oplossing, ik pin een andere versie.